src/components/Invoice.tsx

```diff
@@ -8,108 +8,171 @@ import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import IconButton from '@mui/material/IconButton';
+import { SubmitHandler, useForm, Controller } from 'react-hook-form';
```

Comment:
Not super important here, but I would encourage to use 
```ts
import { type SubmitHandler, useForm, Controller } from 'react-hook-form';
```

instead

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

I encourage to enable the eslint rule which handles that for you. https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md

Reply:
added